docker/gaffer-jhub-options-server/templates/profile_list.pug

```diff
@@ -0,0 +1,150 @@
B5bE3Gxt1bvrlcFFaNBY/qAXIG/Sgoo29hUhrICXpI57JIw17VbXxi8h2olhckmL9IDLknxjiZFQestY/4jsEgvWoh1JwTJt9CLvvfSTSj296dzANq7VNOmB4H9blpx0y/B9UP9WyT/y5ytVpEtTMQae3Pmy07SVX1Bd8z32WmXrd82tRudSqS0b9E5tIIdR/Mfm8n8/DZH6IeyzBzN38UEpJ0J7MZjvJwjvHfpcJCfKlW7kSYNzYJz5UcbnoasNZ+yo9mTlaofyWnsvi6WahWN35h2sWRlrlLbfPjgLX2ZnpIQSV2Mxn7AEdomnXsFTqTumcbYgHU2u8xbSMSWCUay+aMBeBSMOnwsHywnMNOi6h2hqDuw3yT2qAQ6P22M0mDbeX76lv52owLB58CDfACseIRdnsNgpDWgZUbHm112WhCDdCqwN8zSwN51gH3895+M1oAHQs5h5VcYQFaN2W88GANuGwdT3ZxoE5Ia2+Gh3Qrmoz80saKGeBusgsNvug3oBupySKqHe8aaAGp4R7ScOgQZLfEH57C86HBfxFIRu9NbHkKFeIwi1eFEsnucIUdgeiGDDXIzUQagz8Ta15HhTDYQ7oVGspZynDReVPKDQP7MtMuggv2Yp4G8noOLS/83NDip8ElTCydbbfgHV017oKZ4HK+XuzjQE2Q1j57hYG1L4C1n8Ty8IijeKmlYcNiX4Gr1e9EPAC19qdDjKs9FsUdtrEE7tFTPjLoajuBXFUni6h2AlAljJtqDoP9NoWtieYmvKcbxlzsBApfZ+IFqLXfZVTl8RHIc0KYxqEcI5EhM3rh6DAwN27SQK5Ia2+Yh/RfCoOv/ErnLbAbu8CsAjzhxU8cqB/S2hvlIcVn1sWxjYPtQ3hXRqyyPweu88cmxBNQa/+pMd/d8NijCNaSqhIYh5/1v9CPSEbfiviipWLaJMJhlvPq5ZIIvy30XwYT87YJFlHtQK29IbO8u9CgA/tDJD5GBdB9FUz0HnCVrtylgVwHoFleI84j36rwsiquLFlmfGgit0k7V/7iQH4dUHfSiDXFarFFCg/HSBwNNF6rh6xoHR4mHoFa+2IDdizmqb5p+TiqTfWXG6n3u+IgktE/T3wy2CxN82KuhG33xh7SDJbirEPHPw8mXulj0rh9aJnXdOwiGycEl+nnBeo9zb8BzeT24TYNNKo1snYZsApk5ZV7//6pgzZuiM68Zx+JzNP7X98HqttjFL9xII9xZnlIV128s2+5+FK0jSfy991ZsxZxRvcVo+hQn7YexDPdkNb+hiEjhjElG+PjUm3cEbLqUvzaTfr3oD+PNU1Grx8NUPdhXpBNopEwZEttQLwzCOXhgmxOrb2l2ywsotqBWvvAPNmg2rgCqvqsC/9xoIbPoMUp+bJFtbAW1PQZD7IC2EMaLNvUfi4jGf2rZA2g1j5nmmxVe4EW0RfJKkCt/THZrPZ2I5GMvqll0kBOY7jxkDoCX4JiPuRknTiQ+xto/do12bL2gDakvuFOVsJ5IvKQzpRtW302ARn9RGeyFs2Qh/R6uGzd6rIQWUSbkdV4DXUfDsjmrSahyCL6GlmPJ3ncDWE50C6CJy2YhnI8pEWyhavDR7xYRLXTBmntEwplG1edvUhG34asyUDkIU1OlK1cVaa/A/RPA8mqQK39LNnMVSQKKYYakXV5GeVhi2zoKhE0DxTvZQungXqz85BanwvIItrbynGgV5DW/lCMbOvKyUMy+lfI2jyGupNZ0bK1KyMfWUQfI6sDtfZnZHNXQjDyGg8h6wO19umywStkGhLyP+IAaSBPX/BkSQtkk1fEMWQR9XSEOGCtvaEeUsszV+FORq8fUGu/PkS2enlcAxq6Gt7kKHRG3YfVstnLYSbfFlHtDFU4FfjwSPh1UKyhDpQGrLWfXSCbHnEA1Oo5L0eKA9baTwqTba8mjn+LqHZGm+ghtTRFVrCIascHdR+2yea/j8IEUCYfcjz8UR6WygCUIRGd1eHvgGkg13bIQ7pZRuC3zLKKRVQ7/cab5SG1LOiojvH9yDHpizykP4XKFNxlP7KI9iVHpaVZHlJrEnPIEjJ6/ZiA8hAvg/B/ooHjXJngwGnAWvvJuTIKpZwBtRng6shxIL+24JnfT5VZsNnSkUXUjxybjshD+rcomYYFSRa0iGpnBDzmTngP6QpkER1Bjk89lIcMwdMQsh4UpZ4AaSAXpLUP/FnsOKwGNWntIkIcsNb+iNAe0t2gIuM8SAy6IQ/pKYE9pAXIItqNRKEB8pC+JayHNGwSsIg2IHGAWvuzgqYBWkQHCZQGchmGupMlYsZhG6jFMBeR4kAN+yOt/VYR07AUVKJ/QxILqLVfJqCHdDOwiNZ6nERj5IPSQ2orR0Y/ksSjhfSQ2myhU0ERWgiYhnK09ouESkP4dlCCMU4ixoHc26O1tCJ9XkxDJ1y1dycxcUYe0tlfCZOGRPSJ+TtnEhWotZ8syt6LKGR4qfsSiUt31H1QrgixVy8fueiV7iQyUGuvFAsgMk+PQE8+XOg0YK39/zoQmQ6++2L6P+FzW1RGrx9tesK6KCfTHdg9uW5WLHzonm1IdJ5+AOdBSVjqoEPW+cen4Cd+4GmSvKCUx5S/5zncO2PF8rRyn3eIDAOV4yG9m4jzGXvDHGSZ9YatcbOKK3jWXjIKpXgNUCpmyrL119+yNjnJn56cXPFT9nCVUbiz+KGOInnTXQbhF/y6CJ+GLh4yBr/i4St4GnxlGsrMbrYWOg2ta8sIlMHVTeA0+MtepIqAmoKGoWaAbHxAx/5CpqF/R9n0kN5j6woXhrpje8uGL49Woo1A/KmVbPQKGBVQX6Aw1A9wkU1eMbUbNRYkDI0byfdEVboQndsLEIb2AZ6yqauId69aDp2Fia97y0au1jRnM7cODpqFDm7NvGQDVx9nHzff5g6VhOa+bj7OsmHtx6mT94ghTft09XezMv5d+zQdMsKb+5Mw/wuhvxHC8t140gAAAABJRU5ErkJggg==')
```

Comment:
Is there a way we can just store the images somewhere rather than having the actual base64 encoded form in code?

Reply:
The user only directly connects to JupyterHub, so they would have to be hosted there. That would mean creating our own JupyterHub image to add them, which I wanted to avoid.

We could link to locations where they are hosted on the internet, but that would mean they wouldn't load in an air gapped environment.

Reply:
Okay fair enough. Are the images subject to any copyright we might be in trouble for?

Reply:
I've removed the logos for now